evap/staff/importers.py

```diff
@@ -411,9 +408,10 @@ def process(cls, excel_content, semester, vote_start_datetime, vote_end_date, te
 
 
 class UserImporter(ExcelImporter):
-    def read_one_user(self, data):
+    def read_one_user(self, data, sheet, row):
         user_data = UserData(username=data[0], title=data[1], first_name=data[2], last_name=data[3], email=data[4], is_responsible=False)
-        return user_data
+        if user_data not in self.associations.values():
```

Comment:
The check is now earlier. What do you think? The lookup in `.values()` might still be too slow...